src/index.ts

```diff
@@ -69,6 +69,7 @@ export {default as AvatarPair} from './AvatarPair'
 export type {AvatarPairProps} from './AvatarPair'
 export {default as AvatarStack} from './AvatarStack'
 export type {AvatarStackProps} from './AvatarStack'
+export type {BlankslateProps} from './Blankslate'
```

Comment:
We probably don't need this anymore?

Reply:
True, I have forgotten to remove it; Thx,